### PR TITLE
Minor fixes to glossary

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

File: js/glossary.js [hunk 1]
```diff
@@ -1,11 +1,6 @@
...
----
 var glossary = [
-{% for entry in site.data.glossary %}
-{
-"term": {{ entry[0] | jsonify }},
-"def": {{ entry[1] | markdownify | replace:'href="#','href="/glossary/?term=' | jsonify }}
-}{% unless forloop.last %},{% endunless %}
-{% endfor %}
+{%- for entry in site.data.glossary -%}
+    {"term": {{ entry[0] | jsonify }}, "def": {{ entry[1] | markdownify | replace:'href="#','href="/glossary/?term=' | jsonify }}}
```
Comment (thaJeztah) [inline js/glossary.js:3]: We probably don't need to replace the `href="/glossary/?term=`. There's some old script that used this, but looks like it's no longer used. Will have to review that, and can remove in a follow-up

Comment (thaJeztah) [reply js/glossary.js:3]: Follow-up in https://github.com/docker/docker.github.io/pull/11499

**Comment by thaJeztah** [comment]
@usha-mandya @StefanScherer ptal

**Review by StefanScherer — APPROVED** [review]
LGTM

**Pull request merged**